test/e2e/clusterctl_upgrade_test.go

```diff
@@ -61,7 +61,7 @@ var _ = Describe("When testing clusterctl upgrades using ClusterClass (CAPV 1.10
 				InitWithBootstrapProviders:        []string{fmt.Sprintf(providerKubeadmPrefix, capiStableRelease)},
 				InitWithControlPlaneProviders:     []string{fmt.Sprintf(providerKubeadmPrefix, capiStableRelease)},
 				InitWithInfrastructureProviders:   []string{fmt.Sprintf(providerVSpherePrefix, capvStableRelease)},
-				InitWithRuntimeExtensionProviders: []string{},
+				InitWithRuntimeExtensionProviders: testSpecificSettingsGetter().RuntimeExtensionProviders,
```

Comment:
We have to disable `UseKindForManagementCluster: true,` for now.

This is the issue for the storageclass missing in scheme error, because it does not pass through the schema we initialized. It uses the one created [here](https://github.com/kubernetes-sigs/cluster-api/blob/main/test/e2e/clusterctl_upgrade.go#L209) which misses capv relevant stuff.

I will take this as follow up to improve in CAPI.

Reply:
Thanks @chrischdi for the investigation!
So I will disable it and move on. Perhaps add a TODO and enable it after you fix it in CAPI.